start.R

```diff
@@ -241,10 +241,6 @@ if (any(c("--reprepare", "--restart") %in% flags)) {
   # ask for slurmConfig if not specified for every run
   if ("--gamscompile" %in% flags) {
     slurmConfig <- "direct"
-    if (! file.exists("input/source_files.log")) {
-      message("\n### Input data missing, need to compile REMIND first (2 min.)\n")
-      system("Rscript start.R config/tests/scenario_config_compile.csv")
```

Comment:
This line used to trigger the compilation of a testoneregi, right? Why is it deleted now? What was the purpose of the line? Was it to download and distribute input data, and this is done now in `runGamsCompile`?

Reply:
Not exactly, because it sets `action = "c"`, so it compiles a testOneRegi. But yes, the purpose of this line was to download and distribute input data, and this is now done in `runGamsCompile`.

Reply:
Thanks. 
> Not exactly, because it sets action = "c", so it compiles a testOneRegi.
Isn't that what I said? ;-)

Reply:
Ah, yes, one medal for me 🥇